deepinv/physics/mcmri.py

```diff
@@ -0,0 +1,77 @@
+import torch
+from torch import nn
+from deepinv.physics.forward import LinearPhysics
+
+from deepinv.physics import adjoint_function
+
+
+
+
+class MultiCoilMRI(LinearPhysics):
```

Comment:
Can `MultiCoilMRI` inherit from `MRI` to benefit from the mask checking and updating (for on-the-fly mask generation)?

Reply:
(This should be simple enough as the Multicoil MRI is just premultiplying x by the sens maps at forward time, otherwise everything else should be same as MRI)

Reply:
The issue is that MRI has a known SVD but MultiCoil does not.